tools/irc/server.php

```diff
@@ -169,8 +169,8 @@ function server_msg_401($fp, $rdata)
 
 		echo_r('[SERVER_401] ' . $server . ' said: "No such nick/channel" for ' . $nick);
 
-		$db = new SmrMySqlDatabase();
-		$db2 = new SmrMySqlDatabase();
+		$db = MySqlDatabase::getInstance();
+		$db2 = MySqlDatabase::getInstance(true);
```

Comment:
I'd fix this, but I want you to see scrutinizer's error before I do it, and then you wonder why I did it :D

Reply:
Yep, feel free to remove that line.